e2e/helpers/index.js

```diff
@@ -11,7 +11,25 @@ function goToUrl({suite, test, client}) {
   return client.url(getUrl({launchUrl: client.launchUrl, suite, test}));
 }
 
+function formatFileName(testName) {
+  return testName.toLowerCase().replace(/ /g, '-');
+}
+
+function assertVisuals({client, id}) {
+  const isChrome = client.options.desiredCapabilities.browserName === 'chrome';
+  const isMac = client.options.desiredCapabilities.platform === 'macOS 10.12';
```

Comment:
this is very exact version, may not be the same for other machines. Btw, what the Sauce Labs runs on?

Reply:
what do you mean by `may not be the same for other machines`? here we request a chrome under macOS 10.12 - or are you referring to virtualization? if so, I don't know, but I guess that does not matter, does it?